packages/ionic-angular/src/util/mock-providers.ts

```diff
@@ -506,8 +506,9 @@ export function mockTab(parentTabs: Tabs): Tab {
     null
   );
 
```

Comment:
This is also divergent from master. Master has a `overrideLoad` flag that is not included in this mock.